controllers/platformoperator_controller.go

```diff
@@ -125,47 +149,57 @@ func (r *PlatformOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	return ctrl.Result{}, nil
 }
 
-func (r *PlatformOperatorReconciler) ensureBundle(ctx context.Context, po *platformopenshiftiov1alpha1.PlatformOperator, bundleName, bundleImage string) error {
-	if po.Status.InstalledBundleName != "" {
-		return nil
-	}
-	bundle := &rukpakv1alpha1.Bundle{}
-	if err := r.Get(ctx, types.NamespacedName{Name: bundleName}, bundle); err != nil {
+func (r *PlatformOperatorReconciler) ensureBundleInstance(ctx context.Context, po *platformopenshiftiov1alpha1.PlatformOperator, bundle *api.Bundle) error {
```

Comment:
I think we can implement a CreateOrUpdate BundleInstance helper function that mirrors rukpak's CreateOrRecreate logic. This can be done as a follow-up though.

Reply:
Let's handle that in a follow-up for now.

Reply:
https://github.com/timflannagan/platform-operators/issues/10